### ISSUE TITLE
Malformatted CLI output

### ISSUE THREAD

**Issue by TheHolyWaffle** [olistic/warriorjs#111]
![image](https://user-images.githubusercontent.com/787914/40117295-d48f9926-5916-11e8-93ff-23ac7a60035c.png)

## System:
 - OS: Windows 7
 - CPU: x64 Intel(R) Core(TM) i7-6820HQ CPU @ 2.70GHz
 - Memory: 9.43 GB / 15.67 GB
## Binaries:
 - Yarn: 1.6.0 - D:\Program Files\Yarn\bin\yarn.CMD
 - npm: 5.3.0 - C:\Program Files\nodejs\npm.CMD

# Steps to reproduce

Run warriorjs on first stage

# Expected Behavior

Output on terminal should look correct

# Actual Behavior

Output does not look correct

**Comment by olistic** [comment]
Hello @TheHolyWaffle, thanks for reporting this bug. I couldn't reproduce it. I wonder if it has to do with the diamond character next to the score being broken on Windows (for which I created #112).

Now, for me to be able to troubleshoot this, I'd need to know:

* Did this break in level 2 or was also broken in level 1?
* What's the code in `Player.js`?

Thanks!

**Comment by jack45j** [comment]
Same problem here. And I think it broke in every level.
![default](https://user-images.githubusercontent.com/12241340/40134329-a8203d7e-5974-11e8-96d9-c1a4c4a72f6b.png)

**Comment by jack45j** [comment]
I'm using the integrated terminal of visual studio code. 
I'm not in my office right now so I will test the character issue later.

**Comment by Eveneit** [comment]
Ok I've found a working solution, the text is now displaying and animating properly. 
Can someone using Windows test it ?

In `C:\Users\*username*\AppData\Roaming\npm\node_modules\@warriorjs\cli\lib\ui`
or `C:\Users\*username*\AppData\Roaming\npm\lib\ui`

Open printBoard.js and change line 25 :
`const warriorStatusRows = 2;`
to
`const warriorStatusRows = 4;`

![warriorok](https://user-images.githubusercontent.com/5458121/40204189-f1bd76be-5a27-11e8-8f53-3dea7c38f922.png)

**Comment by olistic** [comment]
Hey @Eveneit! Thanks for the efforts you're putting on this. I think I know what's going on thanks to you, and I'm working on a fix. I'll let you know when it's done and, if you could test it, that'd be awesome!

**Comment by olistic** [comment]
@Eveneit the possible fix was pushed to #120.

**Comment by jack45j** [comment]
It works like a charm. 😂

**Comment by olistic** [comment]
@jack45j Which one: @Eveneit's [fix](https://github.com/olistic/warriorjs/issues/111#issuecomment-390014137) or #120? The former is a valid workaround, but is not a fix because it'll break those platforms where this is working correctly. #120 is potentially a fix for everyone, but I'd need confirmation from someone experiencing this issue.

**Comment by jack45j** [comment]
The @Eveneit's fix. I changed `warriorStatusRows` to 4 and the output issue just solved on windows7. But I thought the key point occurs this issue is not caused by this variable.

**Comment by olistic** [comment]
Hey @Eveneit, looking at #134 it looks like #120 fixed this issue. Can you please confirm that so I can close this? Thanks!

**Comment by Eveneit** [comment]
@olistic Yes it did, the game is working alright on Windows since you merged `#print-row-refactor `!
